plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/GraphQLGradlePlugin.kt

```diff
@@ -170,7 +170,7 @@ class GraphQLGradlePlugin : Plugin<Project> {
 
             val configuration = project.configurations.getAt(GENERATE_SDL_CONFIGURATION)
             generateSDLTask.pluginClasspath.setFrom(configuration)
-            generateSDLTask.dependsOn(project.tasks.named("compileKotlin"))
+            generateSDLTask.dependsOn(project.tasks.findByName("compileKotlin") ?: project.tasks.named("compileKotlinJvm"))
```

Comment:
while having explicit dependency on `compileKotlin` task is generally good, I'm thinking whether we should only configure it if it exists, e.g. something along the lines
```suggestion
            val compileKotlinTask = project.tasks.findByName("compileKotlin") ?: project.tasks.findByName("compileKotlinJvm")
            if (compileKotlinTask != null) {
                generateSDLTask.dependsOn(compileKotlinTask)
            } else {
                project.logger.warn("compileKotlin/compileKotlinJvm tasks not found. Unable to auto-configure the generateSDLTask dependency on compile task.")
            }
```